src/main/java/entities/Order.java

```diff
@@ -7,6 +7,21 @@ public class Order extends Item{
     private final String supplier;
     private final int cases;
 
```

Comment:
Added facade design pattern to item class for reading and generating barcodes to delegate barcode tasks to other classes.